composer.json

```diff
@@ -34,21 +34,23 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "~7.4.0 || ~8.0.0 || ~8.1.0",
         "laminas/laminas-cache-storage-implementation": "1.0",
         "laminas/laminas-eventmanager": "^3.4",
         "laminas/laminas-servicemanager": "^3.7",
         "laminas/laminas-stdlib": "^3.6",
+        "lcobucci/clock": "^2.0",
```

Comment:
Let's not depend on one single implementation. `stella-maris/clock` provides an interface implemented by multiple different clock-liberaries. `lcobucci/clock` is one of them.

Reply:
```suggestion
        "stella-maris/clock": "^1.0",
```

Reply:
Discussed on slack: using `stella-maris/clock` is a sufficient solution for now.

1. `lcobucci/clock` implements `stella-maris/clock` for now
2. we can provide our own clock with a minimal anonymous class: `class implements Clock { function now(): DateTimeImmutable { return new DateTimeImmutable(); }}`

This reduces the number of installed dependencies by one.

Also, I didn't know that `lcobucci/clock` required (and implemented) `stella-maris/clock`, at the time of my first review.